cg/store/crud/read.py

```diff
@@ -838,6 +822,18 @@ def get_applications(self) -> list[Application]:
             .all()
         )
 
+    def get_applications_by_ordertype(self, order_type: str) -> list[Application]:
```

Comment:
We could consider a naming which makes it clearer that they are not archived as well, something like `get_active_applications_by_order_type`. Not sure what is better

Reply:
fixed